modules/@apostrophecms/i18n/ui/apos/apps/AposI18nCrossDomainSession.js

```diff
@@ -1,23 +1,5 @@
 export default () => {
-  document.addEventListener('click', e => {
```

Comment:
This turned out to be dead code because of ways the implementation was simplified to do more server-side work, however a small amount of new browser-side work is now required for the cross-domain clipboard because we must bridge LocalStorage between the domains.